assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -98,6 +102,23 @@ function useCanvasKeys(ref) {
       arrangeSelection({ position: getLayerDirection(key, shiftKey) }),
     [arrangeSelection]
   );
+
+  const cloneHandler = () => {
```

Comment:
We should use `useCallback` here, or inline. Otherwise with the `[cloneHandler]` we will be unsubscribing/resubscribing to keyboard event handlers on each render.